src/main/java/com/googleinterns/zoomtube/servlets/DiscussionServlet.java

```diff
@@ -47,8 +48,8 @@
 // TODO: References to Lecture kind string literal need to be updated once Lecture code is merged.
 @WebServlet("/discussion")
 public class DiscussionServlet extends HttpServlet {
-  private static final String PARAM_LECTURE = "lecture";
-  private static final String PARAM_PARENT = "parent";
+  @VisibleForTesting protected static final String PARAM_LECTURE = "lecture";
```

Comment:
why don't we just make it public instead? Since it is used outside of this class. We should be conservative with @VisibleForTesting and find other options before opting for it.

Reply:
Changed

Reply:
@f-camp isn't making it public because of tests less ideal than @VisibleForTesting?